platforms/documentation/docs/src/snippets/testKit/testKitFunctionalTestSpockBuildCache/groovy/src/test/groovy/org/gradle/sample/BuildLogicFunctionalTest.groovy

```diff
@@ -18,7 +18,7 @@ class BuildLogicFunctionalTest extends Specification {
         buildFile = new File(testProjectDir,'settings.gradle') << """
             buildCache {
                 local {
-                    directory '${localBuildCacheDirectory.toURI()}'
+                    directory = file('${localBuildCacheDirectory.toURI()}')
```

Comment:
:x: What? This is already a `File`, we can just:

```suggestion
                    directory = localBuildCacheDirectory
```

Reply:
This is a plain string.